test/infamy/multicast.py

```diff
@@ -0,0 +1,56 @@
+import time
+import subprocess
+import signal
+import sys
+
+class MCastSender:
+    def __init__(self, netns,group):
+        self.group = group
+        self.netns = netns
+
+    def __enter__(self):
+        cmd = f"msend -I iface -g {self.group}"
+        arg = cmd.split(" ")
+
+        self.proc = self.netns.popen(arg, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    def __exit__(self, _, __, ___):
```

Comment:
Yes @wkz, it will: The documentation for [`__exit__`](https://docs.python.org/3/reference/datamodel.html#object.__exit__) tells us that as long as no truthy value is returned, then the exception will propagate upwards as normal.

Reply:
Thank you @wkz. Very nice of you to look that up!